src/Item.php

```diff
@@ -21,6 +21,14 @@ public function getTaxRate()
         return $this->getParameter('tax_rate');
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function getTotalAmount()
+    {
+        return $this->getParameter('total_amount');
```

Comment:
Please add these functions to the relevant unit test (`ItemTest::testGetters`) as well